### PR TITLE
MAINT: Correct deps

### LINKED ISSUE #25: Deps need updating
The list of dependencies  must be updated for 0.11 and the build number should be bumped

### PR DESCRIPTION
**Pull request by larsoner**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

@hoechenberger this was the only error I found looking at https://mne.tools/mne-bids/dev/install.html, was this what you were thinkin, did I miss something, or does the HTML need updating?

Closes https://github.com/conda-forge/mne-bids-feedstock/issues/25

### CODE REVIEW

**Comment by larsoner** [comment]
@conda-forge-admin, please rerender

**Comment by larsoner** [comment]
... right, and `importlib_resources`. I've removed that. Feel free to merge if this is what you had in mind @hoechenberger

**Review by sappelhoff — APPROVED** [review]
in https://github.com/conda-forge/mne-bids-feedstock/pull/24#issue-1402052396 it also complained about pymatreader and edflib-python ... but no idea why. Perhaps only because we never explicitly import them in mne-bids and they are only needed in mne-python?

Anyhow, this lgtm - thanks

**Comment by hoechenberger** [comment]
@larsoner I thought we don't support MNE-Python before 1.0 (or even 1.1???) anymore either, no?

**Comment by hoechenberger** [comment]
and thanks for looking into this, @larsoner!!!

**Comment by sappelhoff** [comment]
ah, right. we only support mne >= 1.0 now

**Comment by hoechenberger** [comment]
> in [#24 (comment)](https://github.com/conda-forge/mne-bids-feedstock/pull/24#issue-1402052396) it also complained about pymatreader and edflib-python

I found these heuristics to be sometimes misleading in the past in other feedstocks as well, so I wouldn't worry to much about them!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,12 +23,11 @@ requirements:
   run:
     - python >=3.7
     - setuptools >=46.4.0
-    - importlib_resources  # this is specific to conda-forge-based installs
     - mne-base >=0.24
```
Comment (hoechenberger) [inline recipe/meta.yaml]:
```suggestion
    - mne-base >=1.0
```

**Comment by hoechenberger** [comment]
I think we're good now

**Comment by larsoner** [comment]
> in https://github.com/conda-forge/mne-bids-feedstock/pull/24#issue-1402052396 it also complained about pymatreader and edflib-python ... but no idea why. Perhaps only because we never explicitly import them in mne-bids and they are only needed in mne-python?

Yes this would make sense if you make use of `mne.export` because the deps are via `mne` in that case

Thanks for the quick responses, fix, and marking for auto-merge!

**Pull request merged**